ndmg/register/gen_reg.py

```diff
@@ -657,7 +657,7 @@ def atlas2t1w2dwi_align(self, atlas, dsn=True):
             self.atlas_img = nib.load(self.dwi_aligned_atlas)
         else:
             self.atlas_img = nib.load(self.aligned_atlas_t1mni)
-        self.atlas_data = self.atlas_img.get_data().astype("int")
+        self.atlas_data = np.around(self.atlas_img.get_data()).astype("int16")
```

Comment:
Why np.around now?

Reply:
In this case, rounding ensures that atlas intensities of 0.999 are interpreted as 1 and not 0.